test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+# The ONBUILD strategy only works with the application source dir so we need
+# to manually specify to copy to another location.
+#
+# This is a little hack-ish given that we know our assemble script requires files to be in /tmp/src
+# we will copy there, and also set our WORKDIR to be the same location so it has access to the scripts
+ONBUILD COPY . /tmp/src/
```

Comment:
per our IRC discussion i'm accepting temporary defeat here.  but we should remove the S2I-OnBuild assumption that requires the images contain a COPY command of this form.  We should just inject the source into the image ourselves, like we do in the normal s2i flow.